### PR TITLE
Bypass Test cases for Trexle.

### PR DESCRIPTION
**Pull request by vikramjadon921995**

### CODE REVIEW

File: test/gateways/trexle_test.exs [hunk 1]
```diff
@@ -54,6 +55,91 @@ defmodule Gringotts.Gateways.TrexleTest do
     description: "For our valued customer, Mr. Potter"
   ]
 
+  @invalid_amount_response ~s/{"error":"Payment failed","detail":"Amount must be at least 50 cents"}/
+  @invalid_card_response ~s/{"error":"Payment failed","detail":"Your card's expiration year is invalid."}/
+  @valid_request_response ~s/{"response":{"token":"charge_3e89c6f073606ac1efe62e76e22dd7885441dc72","success":true,"captured":false}}/
+
+  describe "core" do
```
Comment (oyeb) [inline test/gateways/trexle_test.exs]: Why is this `describe` block called "core"?

File: lib/gringotts/gateways/trexle.ex [hunk 2]
```diff
@@ -340,6 +339,13 @@ defmodule Gringotts.Gateways.Trexle do
...
 
+  defp respond({:ok, %{status_code: code, body: _body}}) when code in [401] do
+    {
+      :error,
+      %Response{reason: "Unauthorized access.", message: "Unauthorized access"}
+    }
+  end
+
   defp respond({:ok, %{status_code: status_code, body: body}}) do
     {:ok, results} = decode(body)
```
Comment (oyeb) [inline lib/gringotts/gateways/trexle.ex]: So what does trexle send in the response body for 401? Is there some JSON?

Comment (vikramjadon921995) [reply lib/gringotts/gateways/trexle.ex]: This is the response:

File: test/gateways/trexle_test.exs (hunk 1 above)
Comment (vikramjadon921995) [reply test/gateways/trexle_test.exs]: I'll change it and name it `Authorize integration point testing`

**Pull request closed without merging**